cypress/e2e/shared/firstmile.tests.ts

```diff
@@ -21,64 +20,64 @@ describe('First mile experience', () => {
       cy.contains('Hello, Time-Series World!')
 
       // second page is install dependencies
-      cy.contains('Next').click()
```

Comment:
what's the intent behind changing this from `contains('Next')` to `getByTestID('python-next-button')`?

- The `contains('Next')` was deliberate - it's more akin to a user's behavior than a machine's behavior, and it tests behavior more than implementation. The user will click the Next button. A machine will find an element in the DOM by test id and click the button matching that. The implementation of the feature is to have a specific test id, whereas the behavior of the feature is to advance the page when a user clicks Next. By targeting test ids when we could target the Next button, we more tightly couple the test to the implementation of the DOM.

Reply:
So what ended up happening was there is one page in the CLI onboarding that has text somewhere along the lines of `Next, we will write data` or similar, so the test tried clicking on that instead of the `Next` button, causing it to fail every time on that page, so I changed it on every section where it clicks on that button.

Reply:
Big sad